content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -49,89 +38,231 @@ Click the :guilabel:`Quotation Templates` link on the :guilabel:`Settings` page,
+from a variety of pre-configured amounts of time (e.g. :guilabel:`Monthly`, :guilabel:`Quarterly`)
+to designate how often this quotation template should occur.
+
+.. note::
+   The :guilabel:`Recurrence` field **only** applies to subscription plans. For more information,
+   check out the documentation on :doc:`/applications/sales/subscriptions/plans`.
+
+Lines tab
+---------
 
 In the :guilabel:`Lines` tab, products can be added to the quotation template by clicking
-:guilabel:`Add a product`, organized by clicking `Add a section` (and dragging/dropping section
-headers accordingly), and further explained with discretionary information (such as warranty
-details, terms, etc.) by clicking :guilabel:`Add a note`.
+:guilabel:`Add a product`, organized by clicking :guilabel:`Add a section` (and dragging/dropping
+section headers), and further explained with discretionary information (such as warranty details,
+terms, etc.) by clicking :guilabel:`Add a note`.
+
+.. image:: quote_template/lines-tab-quotation-template.png
+   :align: center
+   :alt: Populated lines tab on a quotation template form in Odoo Sales.
+
+To add a product to a quotation template, click :guilabel:`Add a product` in the :guilabel:`Lines`
+tab of a quotation template form. Doing so reveals a blank field in the :guilabel:`Product` column.
+
+When clicked, a drop-down menu with products from the database appear. Select the desired product
+from the drop-down menu to add it to the quotation template.
+
+.. tip::
+   If the desired product isn't readily visible, type the name of the desired product in the
+   :guilabel:`Product` field, and the option appears in the drop-down menu. Products can also be
+   found by clicking :guilabel:`Search More...` from the drop-down menu.
+
+.. note::
+   When a product is added to a quotation template, the default :guilabel:`Quantity` is `1`, but
+   that can be edited at any time.
+
+Then, drag-and-drop the product to the desired position, via the :guilabel:`six squares` icon,
+located to the left of each line item.
+
+To add a *section*, which serves as a header to organize the lines of a sales order, click
+:guilabel:`Add a section` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in
+which the desired name of the section can be typed. When the name has been entered, click away to
+secure the section name.
+
+Then, drag-and-drop the section name to the desired position, via the :guilabel:`six squares` icon,
+located to the left of each line item.
+
+To add a note, which would appear as a piece of text for the customer on the quotation, click
+:guilabel:`Add a note` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in which
+the desired note can be typed. When the note has been entered, click away to secure the note.
+
+Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon, located
+to the left of each line item.
```

Comment:
```suggestion
Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon.
```
I feel like we've been told where it is a few times now, so to avoid redundancy and keep it short/sweet, I deleted it here. But, it's just a suggestion so ignore if you'd like to keep it